java/bench/src/java/org/apache/orc/bench/CompressionKind.java

```diff
@@ -53,6 +54,8 @@ public OutputStream create(OutputStream out) throws IOException {
         return new GZIPOutputStream(out);
       case SNAPPY:
         return new SnappyCodec().createOutputStream(out);
+      case ISAL:
```

Comment:
ISAL is just the library for gzip? That doesn't need a new codec - have you tried LD_PRELOAD to load ISAL instead of libz?